test/manual/docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3'
```

Comment:
why do you think it is needed to keep in the repo if in the end it is not used in any automated tests? what is the value of this file?

Reply:
It seems current image is not tested at all, not automated nor manual. Example: the ARG for cli version that I removed as dysfunctional. I guess that behavior has been around for many months.
Than a manual test is better than nothing. 
I lack skill to develop an automated test for the image, but fine with me to drop the manual test resource and raise an issue to validate the docker image in some automated way.

Reply:
On the ARG... I have pulled image from DockerHub and inspected version of asyncapi in resulting container. The ARG is ignored in building of image! Current version is 1 something, not 0.59 as requested per ARG. And this is good. 
I can see value to manually install an older (trusted) version of CLI, but for the image you would expect latest and greatest version. Docker Hub has the older versions as well as in `docker pull asyncapi/cli:1.6.14`

Reply:
oh, you're right, we missed it and did not add a simple docker build testing - lemme fix that, we have some generic workflows that I can pull in

Reply:
ok, we have now https://github.com/asyncapi/cli/blob/master/.github/workflows/if-docker-pr-testing.yml in this repo, lets update this PR

Reply:
ok, so automated tests are in place now

next

yes, we know the arg is ignored (overwritten) in release: https://github.com/asyncapi/cli/blob/master/.github/workflows/release-docker.yml#L42 so we always want to release a latest

the arg in docker file was added for a reason -> see the story in https://github.com/asyncapi/cli/pull/675

does it explain all? 
maybe instead of removing `ARG ASYNCAPI_CLI_VERSION=0.59.1` we should add a comment for future, linking to the other PR I pointed out?

Reply:
You predict my thoughts. The ARG goes back in with comment that it is for manual image create only, github workflow overrides. #675 has lots of insider communication, perhaps a bit too much ;-)